endpoints/openrtb2/video_auction.go

```diff
@@ -440,15 +440,15 @@ func getVideoStoredRequestId(request []byte) (string, error) {
 
 func mergeData(videoRequest *openrtb_ext.BidRequestVideo, bidRequest *openrtb.BidRequest) error {
 
-	if videoRequest.Site.Page != "" {
-		bidRequest.Site = &videoRequest.Site
+	if videoRequest.Site != nil {
+		bidRequest.Site = videoRequest.Site
```

Comment:
Do we still want fo check for the `Page` string content?
```
441 func mergeData(videoRequest *openrtb_ext.BidRequestVideo, bidRequest *openrtb.BidRequest) error {
442
443     if videoRequest.Site != nil && videoRequest.Site.Page != "" {
444         bidRequest.Site = videoRequest.Site
445         if &videoRequest.Content != nil {
446             bidRequest.Site.Content = &videoRequest.Content
447         }
video_auction.go
```

Reply:
@guscarreon thank you for your comment. 
According to Open RTB specification this is the definition of `page` parameter: 
```
Attribute | Type    | Description
page       | string   | URL of the page where the impression will be shown.
```
I can add reg exp verification if needed. 
@hhhjort  please advice
 